libs/structwalk/walktype.go

```diff
@@ -14,6 +14,9 @@ import (
 //   typ          the field's type – if the field is a pointer to a scalar the pointer type is preserved;
 //                the callback receives the actual type (e.g., *string, *int, etc.).
 //
+// The function returns a boolean:
+//   skip: if true, the WalkType function will skip walking the current field and all its children.
```

Comment:
can we use positive name here to avoid double negative for the common case? can be named continue or process or enter.